test/serving.bash

```diff
@@ -62,6 +63,7 @@ function upstream_knative_serving_e2e_and_conformance_tests {
     ./test/e2e ./test/conformance/api/... ./test/conformance/runtime/... \
     --resolvabledomain --kubeconfig "$KUBECONFIG" \
     --imagetemplate "$image_template" \
+    --customdomain="$subdomain"
```

Comment:
I guess this line is missing the backslash.

Reply:
Ah! Thank you!